docs/setup/1-tezos-client.md

```diff
@@ -117,22 +140,27 @@ Error:
 > tutorials to avoid this error. If you use your own local Tezos node you can
 > omit `--wait none`, the error won't happen.
 
-Make a variable for Alice's account address (notice that the address
+Make a variable for Alice's account address  (PHK) (notice that the address
```

Comment:
It's PKH (public key hash), not PHK. Introduction of this acronym here and without explanation doesn't seem very useful